### PR TITLE
initialized, added dependencies

### PR DESCRIPTION
**Pull request by harjothkhara**

### CODE REVIEW

File: data/helpers/dishes-model.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+async function update(item, id) {
+  const editedDish = await db("dishes")
+    .where({ id })
+    .update(item);
+  if (editedDish) {
+    const dish = await findById(id);
+    return dish;
+  }
+}
```
Comment (angel-torres) [inline data/helpers/dishes-model.js]: ✔️

File: data/helpers/ingred-model.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+async function update(item, id) {
+    const editedIngredient = await db("ingredients")
+        .where({ id })
+        .update(item);
+     if(editedIngredient) {
+         const ingredient = await findById(id);
+         return ingredient;
+     }   
+}
```
Comment (angel-torres) [inline data/helpers/ingred-model.js]: I'm not sure you need to put `async` before each function. I think you only need it when you call the function not when you define it.

File: data/migrations/20190606141206_lambda.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  });
+};
+
+exports.down = function(knex, Promise) {
+return knex.schema
+  .dropTableIfExists("recipe_ingredients")
+  .dropTableIfExists("ingredients")
+  .dropTableIfExists("recipes")
+  .dropTableIfExists("dishes");
+};
```
Comment (angel-torres) [inline data/migrations/20190606141206_lambda.js]: ✔️

File: routers/dishRouter.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            }
+        } catch(error) {
+            res.status(500).json({
+                message: `The dish's information could not be modified: ${error}.`
+            });
+        }
+    }   
+});
+
+module.exports = router;
```
Comment (angel-torres) [inline routers/dishRouter.js]: Api looks good! 👍

File: routers/ingredRouter.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        } catch(error) {
+            res.status(500).json({
+                message: `The ingredients information could not be modified: ${error}.`
+            });
+        }
+    }   
+});
+
+module.exports = router;
```
Comment (angel-torres) [inline routers/ingredRouter.js]: ✔️

File: routers/recipeRouter.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+            }
+        } catch(error) {
+            res.status(500).json({
+                message: `The recipe's information could not be modified: ${error}.`
+            });
+        }
+    }   
+});
+
+module.exports = router;
```
Comment (angel-torres) [inline routers/recipeRouter.js]: ✔️

File: server.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const helmet = require("helmet");
+const dishRouter = require("./routers/dishRouter.js");
+const recipeRouter = require("./routers/recipeRouter.js");
+const ingredRouter = require("./routers/ingredRouter.js");
+
+const server = express();
+
+//middleware
+server.use(helmet());
```
Comment (angel-torres) [inline server.js]: Don't forget to add `cors`! You'll definitely need it for build week.

**Review by angeltorgon — COMMENTED** [review]
## Minimum Viable Product Objectives
* [x] Design the data model for a `recipe book` application
* [x] Use `Knex migrations and seeding` functionality to build a `SQLite3` database based on the model and seed it with test data
* [x] Write a data access file 

## Stretch Objectives
* [x] Build RESTful API that makes use of your data access file
* [x] Add a method called `getRecipe(id)` to your data access library
* [x] Add the CRUD operations for other entities in the system+
* [ ] Design a front end app that consumes your API

# Project Manager Feedback
## Great
* You completed all MVP objectives! :heavy_check_mark:
* You completed stretch objectives! :heavy_check_mark:
Awesome job Harjoth!
## Requested Improvements
* Build a react app that consumes your api!
## Questions
* How comfortable to you feel using knex to create and design a database?
## Rating {3}
:star: :star: :star:

**Pull request merged**